### PR TITLE
Audit spawns of tire-changing equipment in vehicles

### LINKED ISSUE #56881: Tire Irons should spawn more, and also be renamed to Lug Wrenches
### Is your feature request related to a problem? Please describe.

Tire irons are required to change a car's tires, but seem to be quite rare. According to the HHG (https://nornagon.github.io/cdda-guide/#/item/tire_iron), they have only a 31% chance of spawning in garages! Changing a car's tires is one of the most common tasks an auto shop does, and they should have tire irons on hand--even assuming some or most have been looted by the time the player arrives. Tire irons should also spawn frequently in car trunks; many cars have a lug wrench in the trunk to go along with the spare tire.

Also, what the game calls a "tire iron" is actually a lug wrench, so it should be renamed accordingly.
Tire iron: https://en.wikipedia.org/wiki/Tire_iron
Lug wrench: https://en.wikipedia.org/wiki/Lug_wrench

### Solution you would like.

Rename the Tire Iron to Lug Wrench.

Lug wrenches should spawn frequently in garages and car trunks, and less frequently (but still more than their present rarity!) in other locations where tires are changed, such as residential garages, rural homes and farms, and military bases.

### Describe alternatives you have considered.

Renaming tire irons isn't totally necessary.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Davi-DeGanne**
#### Summary
Balance "Audit spawns of tire-changing equipment in vehicles"

#### Purpose of change

(Story time! TL;DR further down)

So my mid-late game survivor (let's call the poor sap "Jim") steps out of his base of operations one day, and decides it's finally time for a vehicle. He walks to the nearby town which has been mostly cleared of "unruly neighbors", and picks out a fixer-upper of a Humvee. After some diagnostics, he makes a short list of tools that he doesn't have: Cordless drill to mount a new gas tank, and a bottle jack and a tire iron to change the tires.

Ah, the tire iron (lug wrench for the semantic professional). The tool that lets even the most clueless car-owner claim mechanical literacy, you'd be hard-pressed to walk down a residential block and not find a half dozen cars with one of these esteemed bad boys. So my survivor tries exactly that. Hmmm... no luck. Whole town was full of people who figured they could change a tire with a scissor jack and a dream, apparently. No matter, Jim thought, I have an advanced forge setup and the skills to use it. However, here too, Jim's plans were foiled, because blueprints for the enigmatic tire iron eluded his grasp. "Truly, this is a tool of the gods, unforgeable by the feeble hands of mere men!" Jim cried.

So began the two-day quest through a world filled with megacities to find the holy tire iron. Battles were fought, small victories were had, and injuries mounted slowly but inexorably (primarily thanks to damn shockers and grapplers, both of whom eat fools trying to swoosh by on electric bikes for breakfast). Ultimately, Jim was forced to return home in shame, with a tire-iron-shaped hole in his heart.

Anyway, in my moderately contained fury at this absurd situation, I searched the save files, and turns out, despite having explored a total of nearly 5,000 map tiles, there did not exist a single tire iron anywhere in the entire game world. So, as any perfectly sane human being would do, I channeled my fury into a glorious PR.

TL;DR:
- Fixes #56881
- Addresses general inconsistencies and lack of realism with how jacks, tires, and tire irons spawn in vehicles (and as a bonus, in the `mx_mayhem` map extra).
- Renames "tire iron" to "lug wrench" which I think is good to get out of the way early, so it's already obsoleted by the time someone decides to add _actual_ tire irons. (Note that I use them somewhat interchangeably in this PR description; I'm hoping that cures more confusion than it causes, but ¯\\_(ツ)_/¯ )

#### Describe the solution
- First, held a survey to help come up with accurate numbers:
  - 35 responders had at least one
  - 26 had a tire (~74%)
  - 32 had a jack (~92%)
  - 31 had a lug wrench (~88%)
- Split tires, and tire-changing gear, away from `car_kit`, creating a new item group `spare_tire_kit`, which was added to vehicles it made sense for; for a lot of these, that also meant deleting existing in-line spawns of tires and jacks (but not tire irons). Overall, though, all three items will be vastly more common.
- I decided on the following modifiers (additive) to determine the spawn chance of `spare_tire_kit` (these are based on rough googling and are very much open to suggestions; the one really credible data point I found was a AAA study saying that only ~30% of new cars sold today come with a spare tire kit and tools):
  - Baseline                         40%
  - Hybrid vehicle:                 -15%
  - Electric vehicle:               -30%
  - SUV:                            +40%
  - Pickup/van/bus/RV:              +60%
  - Sports vehicle:                 -30%
  - Part of organizational fleet:  Never
  - Budget vehicle:                Never
  - Police cruisers:              Always
  - Exceptions:
    - 4x4 has 100% chance, but gets a modified spawn group with no tire (it's already mounted externally).
    - Survivor vehicles always spawn with tools, but sometimes no tire.
    - Hatchbacks should have 40% but don't have a cargo space big enough for a tire, so the spare tire and jack were removed entirely.
- Spawn chances of car_kit were lowered by 1/3 so the spawn rates of other items therein should be unchanged.

#### Describe alternatives you've considered
I thought about adding an under-frame spare tire mount, which would be much more realistic for pickup trucks and vans and such, but I didn't want to feature-creep this PR so much that I never published it.

I waffled quite a bit on the spawn rates of the spare tire kits in different types of cars.

I wish I could think of some way to solve the hatchback problem.

Should I have extended my find/replace of tire_iron -> lug_wrench to the gfx folder? Or is that fully managed in the other repo...?

#### Testing
- Spawned in `mx_mayhem` map extra until I got the tire-change variant. Dude spawns with a scissor jack now.
- Went through some vehicles in town. Tire irons (lug wrenches) are much more common now, and the three pieces (wrench+jack+tire) frequently, but not always, spawn together.
- Loaded up an old save with tire iron in the game world, migration went through great with no error messages.

#### Additional context
I seem to have spent the better part of my waking hours today obsessively researching and designing simulation parameters for spare tires in cars and other larger vehicles, despite the fact that I don't, and may never, own a vehicle with more than two wheels. Ah, Cataclysm, I've missed you

### CODE REVIEW

**Comment by Davi-DeGanne** [comment]
That should appease clang, lgtm failure was just a timeout so hopefully it's fine

File: data/json/items/tool/workshop.json [hunk 1]
```diff
@@ -1164,9 +1164,9 @@
     ]
   },
   {
-    "id": "tire_iron",
+    "id": "lug_wrench",
```
Comment (dseguin) [inline data/json/items/tool/workshop.json:1167]: I understand changing the item's translated name, but should the ID really be changed? It's not user-facing data, and some 3rd party mods might be relying of the ID. Not sure about this, but feel free to correct me

Comment (Davi-DeGanne) [reply data/json/items/tool/workshop.json:1167]: I went back and forth on whether to change the name at all; the thing that pushed me over the edge was that I wouldn't be surprised if someone wanted to add **actual** tire irons within the year. If they do, what ID are they going to use? Sure, they can just use `tire_level` or `tire_spoon` or `tire_iron_actual` or something, but if this is still tire_iron when they do, doesn't it seem likely that future coders will conflate the two items, possibly even creating more bugs in the process? I know these are all hypotheticals but imo these things are important to nip in the bud early, before they have a chance to accrue more technical debt.

If the mergers agree with you and not me, though, I will happily concede this point. There's even a third option, which is to add a duplicate `tire_iron` item to `obsolete.json`.

Comment (dseguin) [reply data/json/items/tool/workshop.json:1167]: That makes sense. Cheers!

Comment (Davi-DeGanne) [reply data/json/items/tool/workshop.json:1167]: For sure 😄 thanks for the review!
